db/migrate/20201216184415_create_rejected_contacts.rb

```diff
@@ -2,11 +2,11 @@ class CreateRejectedContacts < ActiveRecord::Migration[6.0]
   def change
     create_table :rejected_contacts do |t|
       t.references :imported_item, null: false, foreign_key: true
+      t.string :reason
       t.string :test_and_trace_account_id
       t.string :nhs_number
       t.boolean :is_vulnerable
       t.string :first_name
-      t.string :middle_names
```

Comment:
Shouldn't really be changing migrations after pushing as it breaks database for anyone who's already run - should be a new migration ideally. But given it's not been deployed anywhere, not the end of the world - I can just reset my local database.

Reply:
Agreed @userman123, I just went for fastest solution. Would totally not have done this if it had been deployed anywhere